paddle/api/CMakeLists.txt

```diff
@@ -92,7 +92,9 @@ add_custom_command(OUTPUT ${PROJ_ROOT}/paddle/py_paddle/_swig_paddle.so
 # TODO(yuyang18) : make wheel name calculated by cmake
 add_custom_target(python_api_wheel ALL DEPENDS ${PROJ_ROOT}/paddle/py_paddle/_swig_paddle.so)
 
-install(DIRECTORY ${PROJ_ROOT}/paddle/dist/ DESTINATION opt/paddle/share/wheels)
+install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/dist/
```

Comment:
This wheel package will be generated in `${PROJ_ROOT}/paddle/dist/` automatically. It cannot be controlled by CMake.

After compilation, I checked where is `*.whl`.
gangl@GangLiao ~/b/Paddle> find . -name "*.whl"
`./paddle/dist/py_paddle-0.10.0rc4-py2-none-any.whl`

`install(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/dist/ DESTINATION opt/paddle/share/wheels)`
 means that copy `*.whl` from `${CMAKE_CURRENT_BINARY_DIR}/dist/` to `DESTINATION opt/paddle/share/wheels`

Because `*.whl` does not exist in `${CMAKE_CURRENT_BINARY_DIR}/dist/`, it will cause the compile error...